packages/sns/src/governance.canister.ts

```diff
@@ -104,6 +105,22 @@ export class SnsGovernanceCanister extends Canister<SnsGovernanceService> {
     return proposals;
   };
 
+  /**
+   * List the proposals of the Sns including `include_ballots_by_caller` flag.
+   * When the flag is true, we know that the sns governance canister returns real ballots information.
+   * When the flag is false, there is always an empty list of ballots.
+   */
+  listCallerProposals = async (
```

Comment:
Sorry, but I don't understand the difference between this and `listProposals`.

Isn't the flag in `SnsListProposalsParams` the same as in `listProposals`?

Reply:
Only the return type is different `Promise<ListProposalsResponse>` vs `Promise<ProposalData[]>`, to avoid breaking changes. But not a strong opinion. Would you prefer to change `listProposal` interface?

Reply:
I would add the breaking change and prepare a major version instead of adding a new method.